src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -113,9 +121,9 @@ private void InitializeCore()
                 return;
             }
 
-            _resharperExtensionEnabled = extensionEnabled != 0;
+            _resharperExtensionInstalledAndEnabled = extensionEnabled != 0;
```

Comment:
extensionEnabled seems should be renamed to extensionInstalled?

Reply:
the state feels wrong. at this point, it knows extension is installed or not, but it doesn't know whether extension is enabled or NOT.

so the state (installed && enabled) should be 3 states, false, true, unknown. so using bool looks like not right. if you are going to change the field to present "InstalledAndEnabled", I think you should use bool? and use null as unknown state.

Reply:
Enabled is perhaps an overloaded word here: this enabled refers to the VS concept of an extension being disabled or enabled, not to resharper's internal status.